scripts/get-helm-3

```diff
@@ -33,6 +33,7 @@ initArch() {
     x86_64) ARCH="amd64";;
     i686) ARCH="386";;
     i386) ARCH="386";;
+    s390x) ARCH="s390x";;
```

Comment:
thinking about this more... Why are we checking that `arch == s390x`, then setting it to the same value? The others are necessary because they're translating architectures like `i386` to the expected architecture name `386`, or `x86_64` to `amd64`.

Can you remove this?

Reply:
Thanks for review comments. Removed `s390x arch` check.